src/JsonSchema/Uri/UriResolver.php

```diff
@@ -77,7 +77,11 @@ public function generate(array $components)
     public function resolve($uri, $baseUri = null)
     {
         // treat non-uri base as local file path
-        if (!is_null($baseUri) && !filter_var($baseUri, \FILTER_VALIDATE_URL)) {
+        if (
+            !is_null($baseUri) &&
+            !filter_var($baseUri, \FILTER_VALIDATE_URL) &&
+            !preg_match('|^[^/]+://|u', $baseUri)
```

Comment:
Pipe `|` delimiter so that you do not need to escape the forward slash `/` characters, correct?